xpk.py

```diff
@@ -5722,6 +5783,10 @@ def workload_create(args) -> None:
         node_pool_name=f'{args.cluster}-np-0',
         chips_per_vm=system.chips_per_vm,
         autoprovisioning_args=autoprovisioning_args,
+        gpu_volume=get_gpu_volume(args),
+        gpu_rxdm_image=get_gpu_rxdm_image(args),
+        gpu_rxdm_cmd=get_gpu_rxdm_cmd(args),
+        gpu_tcp_volume=get_gpu_tcp_volume(args)
```

Comment:
I think we can pass system instead of args here? Mainly to have consistency in how we parse the device usage. This will be useful for future refactors